data_processors/pipeline/services/metadata_srv.py

```diff
@@ -114,6 +114,54 @@ def get_wts_metadata_by_subject(subject_id: str) -> List[LabMetadata]:
     )
 
 
+@transaction.atomic
+def get_wgs_metadata_by_subjects(subjects: List[str]) -> List[LabMetadata]:
```

Comment:
So. This is our, so called, "boxed-in function" that, by looking at function name, it hints what it might return. Business rules are boxed-in -- i.e. not offer changes from caller. Better unit test-able and put under radar for any unintended changes.

Down the track; we could add pre-/post-filter after query made, if any. But. Those are not concerned from the caller side. That's the business rule change of this function at service layer.